src/content/pages/account/signup/index.tsx

```diff
@@ -129,9 +130,35 @@ function SignupForm() {
 
         // Don't continue to signup logic if there are errors.
         // We cannot check the useState values, as these will not be updated until the next render.
-        if (fullNameInvalid || usernameInvalid || emailInvalid || passwordInvalid || confirmPasswordInvalid) return
+       if (fullNameInvalid || usernameInvalid || emailInvalid || passwordInvalid || confirmPasswordInvalid) return
+       
+        
 
         // TODO: Signup logic.
+        const asyncSignupCall = async () => {
+            try {
+                const response = await fetch('http://localhost:8000/raw-data/create-user', {
```

Comment:
This should be replaced with `api.get` exported from `api.ts`, which will connected to the specified back-end server and will avoid hard coding an endpoint.

Reply:
Going to unresolve as it seems this was overlooked.